indico/modules/events/papers/controllers/management.py

```diff
@@ -51,3 +54,32 @@ def _process(self):
             logger.info("Paper managers of %r updated by %r", self.event_new, session.user)
             return jsonify_data(flash=False)
         return jsonify_form(form)
+
+
+class RHSwitchReviewingType(RHManagePapersBase):
+
+    CSRF_ENABLED = True
+
+    def _process_PUT(self):
+        event = self.event_new
+        reviewing_type = request.view_args['reviewing_type']
+        if reviewing_type not in ('content', 'layout'):
+            raise Forbidden('Reviewing type not available')
+        event.cfp.toggle_reviewing(reviewing_type=reviewing_type, enable=True)
+        flash(_("Reviewing type enabled"), 'success')
+        logger.info("Reviewing type '{}' for event {} enabled by {}".format(reviewing_type, event, session.user))
```

Comment:
logging always uses `%` and the arguments are passed as a `list, of, args` instead of using `.format()` so the log message passed to the method is static and still contains the actual placeholders.

For event and user  i'd use `%r` instead of `%s` since those are object reprs anyway